### PR TITLE
Add Emitter usage to reactive messaging guides

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: docs/src/main/asciidoc/amqp.adoc [hunk 1]
```diff
@@ -305,6 +305,42 @@ You can build the native executable with:
 ./mvnw package -Pnative
 ----
 
+== Imperative usage
+
+Sometimes, you need to have an imperative way of sending messages.
```
Comment (geoand) [inline docs/src/main/asciidoc/amqp.adoc]:
```suggestion
Sometimes you need to have an imperative way of sending messages.
```

File: docs/src/main/asciidoc/amqp.adoc [hunk 2]
```diff
@@ -305,6 +305,42 @@ You can build the native executable with:
...
+
+For example, if you need to send a message to a stream, from inside a REST endpoint, when receiving a POST request.
```
Comment (geoand) [inline docs/src/main/asciidoc/amqp.adoc]:
```suggestion
For example, if you need to send a message to a stream from inside a REST endpoint, when receiving a POST request.
```

**Comment by loicmathieu** [comment]
@geoand I apply our suggestions

**Comment by geoand** [comment]
Cool, thanks!

File: docs/src/main/asciidoc/amqp.adoc [hunk 3]
```diff
@@ -305,6 +305,42 @@ You can build the native executable with:
 ./mvnw package -Pnative
 ----
 
+== Imperative usage
+
+Sometimes you need to have an imperative way of sending messages.
+
+For example, if you need to send a message to a stream from inside a REST endpoint, when receiving a POST request.
```
Comment (geoand) [inline docs/src/main/asciidoc/amqp.adoc]: Sorry, one more change if I may:

```suggestion
For example, if you need to send a message to a stream from inside a REST endpoint when receiving a POST request.
```

File: docs/src/main/asciidoc/amqp.adoc [hunk 4]
```diff
@@ -305,6 +305,42 @@ You can build the native executable with:
...
+    @POST
+    @Consumes(MediaType.TEXT_PLAIN)
+    public void addPrice(Double price) {
+        priceEmitter.send(price);
+    }
+}
+----
+
+NOTE: The `Emitter` configuration is done the same way as the other stream configuration used by `@Incoming` and `@Outgoing`.
+In addition, you can use `@OnOverflow` to configure back-pressure strategy.
```
Comment (geoand) [inline docs/src/main/asciidoc/amqp.adoc]:
```suggestion
In addition, you can use `@OnOverflow` to configure a back-pressure strategy.
```

**Comment by loicmathieu** [comment]
@geoand I applied your new suggestions

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
LGTM.

Note that the Emitter is slightly changing in the next version (but the part covered in the doc does not change).

**Pull request merged**